install/installer/pkg/config/v1/experimental/experimental.go

```diff
@@ -167,6 +167,16 @@ type WorkspaceTemplates struct {
 	Regular    *corev1.Pod `json:"regular"`
 }
 
+type StripePriceIDs struct {
+	EUR string `json:"eur"`
+	USD string `json:"usd"`
+}
+
+type StripeConfig struct {
+	IndividualUsagePriceIDs StripePriceIDs `json:"individualUsagePriceIds"`
+	TeamUsagePriceIDs       StripePriceIDs `json:"teamUsagePriceIds"`
+}
+
```

Comment:
Hey Milan, I've just joined the team so I'm still catching up on decisions around the installer 😬 

How do we decide if things go to "experimental" or not? maybe @MrSimonEmms could help clarifying as well?

Reply:
@ArthurSens as a general rule, anything that's in `experimental` is for config that's either specific to SaaS or (in rarer cases) to support a specific paying customer. Anything in `experimental` should be considered liable to change without notice.

Things are only added to the main config if it's something that we are happy to publicly support (including community/open source users), we have updated the public documentation (and usually added to the KOTS config) and that we are confident we won't need to amend.

Looking at this, it looks like @easyCZ has correctly put it in `experimental` as this is a SaaS-specific config parameter.

Reply:
Thanks Simon. Yes, indeed this is config only specific to SaaS at this point, hence defined in the experimental section.